cmd/keytransparency-server/frontend.go

```diff
@@ -208,6 +229,7 @@ func main() {
 	}
 	grpcServer := grpc.NewServer(opts...)
 	pb.RegisterKeyTransparencyServiceServer(grpcServer, svr)
+	trillian.RegisterTrillianLogServer(grpcServer, logServer)
```

Comment:
The pros:
- External users of our code won't have to run multiple binaries. 

Cons: 
- A little less process separation, but this can be adjusted by replacing this bit of code with the proxy.